js/queen.js

```diff
@@ -0,0 +1,117 @@
+import { Piece } from "./piece.js"
```

Comment:
Line `1` is missing a semicolon at the end.

Reply:
Added semicolon.